package.json

```diff
@@ -86,7 +86,7 @@
     "bench:server": "node benchmarks/server.js",
     "prebench:run": "node benchmarks/wait.js",
     "bench:run": "SAMPLES=100 CONNECTIONS=50 node benchmarks/benchmark.js",
-    "serve:website": "docsify serve .",
+    "serve:website": "docsify serve documentation",
```

Comment:
Why not creating a package.json for documentatin alone? 

docsify-cli is huuuge and has alot of other deps. Would make node_modules slimmer.

Reply:
Can do